chips/lowrisc/src/hmac.rs

```diff
@@ -0,0 +1,248 @@
+//! HMAC (Hash-based Message Authentication Code).
```

Comment:
which hash algorithm is used?

Reply:
OT only supports SHA256, I have updated the comment